internal/component/database_observability/mysql/collector/explain_plan.go

```diff
@@ -462,6 +460,7 @@ type ExplainPlanArguments struct {
 	PerScrapeRatio  float64
 	EntryHandler    loki.EntryHandler
 	InitialLookback time.Time
+	DBVersion       string
```

Comment:
should we use the same `engine` terminology?

Reply:
Fair play. Updated that as well. We might consider renaming that in the explain plan schema as well, tho that will require frontend changes.